lib/cn/kvset_split.c

```diff
@@ -373,9 +371,8 @@ kblocks_split(
 
         /* TODO: Would it be accurate to use the left and right kblock's hlog here?
```

Comment:
What is this comment Nabeel?

Reply:
The code currently uses the hlog from the source kblock in the left and right kblocks.. I had to TODO in there to confirm if this is fine.. Will check that with Alex.